app/views/books/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= @book.title %>
+highest rating: <%= @book.highest_rating %>
+review author
+lowest rating: <%= @book.lowest_rating %>
+average rating: <%= @book.average_rating %>
```

Comment:
Pay careful attention to the details of the spec. It asked for this information to be in headers.